addon/templates/components/frost-sort-item.hbs

```diff
@@ -1,3 +1,4 @@
+{{frost-select-outlet}}
```

Comment:
This needs to be a unique target name - you don't want other selects targeting this